apps/alpha-test-app/src/components/layout-demo/PageDemo.vue

```diff
@@ -0,0 +1,69 @@
+<script lang="ts" setup>
+import { OnyxHeadline } from "sit-onyx";
+import { FloatingButtonDemo, FlyoutDemo, LayoutSettings, StickyDemo, TooltipDemo } from ".";
+import type { SettingsSections } from ".";
+import { computed } from "vue";
+
+const settings = defineModel<SettingsSections>();
+
+const props = defineProps<{
+  isSidebarOpen: boolean;
+}>();
+const emit = defineEmits<{
+  isSidebarOpen: [value: boolean];
+}>();
+const isSidebarOpen = computed({
```

Comment:
(optional) use `defineModel`

Reply:
can there be more than one v-model?? :O

Reply:
Sure there can be infinite v-models, they just need a different name. If you don't specify a name it will be "modelValue" but you can call it isSidebarOpen:
```ts
defineModel<boolean>("isSideBarOpen")
```

Reply:
amazing, I love it!!! thank youu 😆